Directory.Build.props

```diff
@@ -76,16 +76,16 @@
 				 <Revision>$([System.DateTime]::Now.get_DayOfYear().ToString())</Revision>
 
 				 <!--Version layout schema:
-				80 = Major
-				23 = Minor - Year of release
+				110 = Major
```

Comment:
This should be 100, as it "released" code, and **not ** Beta

Reply:
I think these were forgotten about last year, as this is how it appears in the `alpha` branch. https://github.com/Krypton-Suite/Extended-Toolkit/blob/alpha/Directory.Build.props